File: Python Excel Examples/WorksheetsApiDemo/getWorkSheetWithFormat.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "GetWorkSheetWithFormat.xlsx"
sheetName = "Sheet1"
format = "html"  # Available format: html, pdf, png, jpeg, bmp, emf,tiff
verticalResolution = 0
horizontalResolution = 0
folder = "ExcelDocument"
storage = ""
result = api.get_work_sheet_with_format(name, sheetName, format, vertical_resolution=verticalResolution,horizontal_resolution=horizontalResolution, folder=folder,storage=storage)

File: PHP Excel Examples/WorksheetsApiDemo/getWorkSheetWithFormat.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "GetWorkSheetWithFormat.xlsx";
$sheetName = "Sheet2";
#supports formats:html/pdf/png/jpeg/bmp/emf/tiff
$format = "html";
$verticalResolution = null;
$horizontalResolution = null;
$folder = "input";
$storage= null;
$result = $apiInstance->getWorkSheetWithFormat($name, $sheetName,$format, $verticalResolution,$horizontalResolution,$folder,$storage);